### ISSUE TITLE
Quest Free Knot! is not giving any reputation with Steamwheedle Cartel.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/issues#669]
http://www.wowhead.com/quest=7429/free-knot
As far as I remember it should give like 75 rep with each of the 4 factions, but i might be wrong. Should be more than 0 anyway.

**Comment by lazermon** [comment]
this wayback to nov 1. 2007 claims that the quest "the gordok ogre suit" is what gives the rep, and claim that the rep earned was 250.

https://web.archive.org/web/20071101125215/http://wow.allakhazam.com:80/db/faction.html?wfaction=169

**Comment by Cabro** [comment]
No, check for Free Knot. It also used to give rep, farming the keys was the hard thing since they werent dropping off every 2nd mob as it is now. It used to drop 1-2 per full clear.

**Comment by eron** [comment]
It absolutely gave rep in classic.   I remember very clearly farming rep with keys to free knot.  It's a shame that an error in the Allakhazam guide is perpetuated on Light's Hope.  Can anyone think of a good way to prove this?
